emulator/clients/README.md

```diff
@@ -0,0 +1,23 @@
+# Emulator clients
+
+The emulator has the `--client` command line option, which will initialize a server on the client that listens for incoming connections. The clients transmit data to the server with information on how to set external pins. 
```

Comment:
This needs to be a little more clear. What do you mean by "initialize a server on the client". Do you mean that the emulator opens up an incoming socket where a client can connect and send it events it should apply to the pins of the emulated  FlexPRET?

Reply:
Yes, that is exactly what I mean. I added it now :)